lib/rules/prefer-arrow-callback.js

```diff
@@ -281,11 +281,11 @@ module.exports = {
                             if (callbackInfo.isLexicalThis) {
 
                                 // If the callback function has `.bind(this)`, replace it with an arrow function and remove the binding.
-                                return fixer.replaceText(node.parent.parent, asyncKeyword + paramsFullText + " => " + sourceCode.getText(node.body));
+                                return fixer.replaceText(node.parent.parent, `${asyncKeyword + paramsFullText} => ${sourceCode.getText(node.body)}`);
                             }
 
                             // Otherwise, only replace the `function` keyword and parameters with the arrow function parameters.
-                            return fixer.replaceTextRange([node.start, node.body.start], asyncKeyword + paramsFullText + " => ");
+                            return fixer.replaceTextRange([node.start, node.body.start], `${asyncKeyword + paramsFullText} => `);
```

Comment:
Nit: Can we change `${asyncKeyword + paramsFullText}` to `${asyncKeyword}${paramsFullText}`? The fixer couldn't do this because it would cause a behavior change if both variables were numbers, but in this case it's not a problem because both variables are strings.